build.gradle.kts

```diff
@@ -0,0 +1,99 @@
+plugins {
+  id("java")
+}
+
+group "com.argv-transform" // Replace with your group name
```

Comment:
Is this the correct name?

Reply:
great question. As you can probably tell that part came from a template and needs to be fixed. Do you know the proper name?

Reply:
@psybers @esherman77 can you tell me what the correct name for the group is?

Reply:
If you can have hyphens, probably something like `dev.arg-v.transformer`.